### ISSUE TITLE
Integrity check fails

### ISSUE THREAD

**Issue by raskyer** [telegraf/telegraf#1678]
It seems there is an integrity check issue with typegram : 
```
 Running 'npm install --unsafe-perm'...
    
 npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
 npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
 npm WARN deprecated rollup-plugin-inject@3.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.
 npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
 npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
 npm WARN tarball tarball data for typegram@git+ssh://git@github.com/MKRhere/typegram.git#e2ba9f01f14b96c6dbb1c617e66692a3cc776cf6 (sha512-mLZ+OTcYqMY5QMLkXXr8Ssx3PZOJEFMy01maedKGnRIJBw0eLPfiQxA4WRQrsYx+YzFbqj+mDxhyIEkYn7MLgA==) seems to be corrupted. Trying again.
 npm WARN tarball tarball data for typegram@git+ssh://git@github.com/MKRhere/typegram.git#e2ba9f01f14b96c6dbb1c617e66692a3cc776cf6 (sha512-mLZ+OTcYqMY5QMLkXXr8Ssx3PZOJEFMy01maedKGnRIJBw0eLPfiQxA4WRQrsYx+YzFbqj+mDxhyIEkYn7MLgA==) seems to be corrupted. Trying again.
 npm ERR! code EINTEGRITY
 npm ERR! sha512-mLZ+OTcYqMY5QMLkXXr8Ssx3PZOJEFMy01maedKGnRIJBw0eLPfiQxA4WRQrsYx+YzFbqj+mDxhyIEkYn7MLgA== integrity checksum failed when using sha512: wanted sha512-mLZ+OTcYqMY5QMLkXXr8Ssx3PZOJEFMy01maedKGnRIJBw0eLPfiQxA4WRQrsYx+YzFbqj+mDxhyIEkYn7MLgA== but got sha512-BcD3DFa5Zrs4X2bEya5YnZE2KBhERsBmBVZsgpV5wzpIUvcR6d5rV9VfXA4Cjd/O5O9WTjeJUMHwE7RKnXmtjw==. (48855 bytes)
```

```
npm WARN skipping integrity check for git dependency ssh://git@github.com/MKRhere/typegram.git
```

Using Telegraf version :
4.9.0

Using Node version:
v16.14.2

Using Npm version:
8.5.0

**Comment by raskyer** [comment]
In order to be able to deploy on Azure, I was force to go back to 4.8.9 and force typegram version.

**Comment by raskyer** [comment]
I believe @MKRhere this line is responsible : https://github.com/telegraf/telegraf/blob/v4/package.json#L71

**Comment by MKRhere** [comment]
typegram upstream won't be updated for another week or two, and npm is unable to integrity check modules installed from Github. You can either disable strict integrity checks or stay on an older release until typegram is updated.

**Comment by MKRhere** [comment]
typegram has been updated to [v3.11.0](https://www.npmjs.com/package/typegram/v/3.11.0)
Fixed in e1acda0b08b34c3ecc989035014222f3eb23467c, released as telegraf [v4.9.1](https://www.npmjs.com/package/telegraf/v/4.9.1)
